jose/test/src/black/door/jose/jws/CirceJwsSpec.scala

```diff
@@ -0,0 +1,5 @@
+package black.door.jose.jws
+
+import black.door.jose.json.circe.JsonSupport
+
+class CirceJwsSpec extends JwsSpec with JsonSupport
```

Comment:
Can we use `circe.JsonSupport` here just so that we can see at a glance that we haven't accidentally imported the wrong json library for these tests?

Reply:
Yea, I have changed it 👍